From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
Date: Wed, 16 Aug 2017 20:17:57 +0300
Subject: [ARCHEOLOGY] [WIP] Initial Rock64 support

> X-Git-Archeology: - Revision 43b3beed8b35cf4ea325068cf5a07dcfd33d98ec: https://github.com/armbian/build/commit/43b3beed8b35cf4ea325068cf5a07dcfd33d98ec
> X-Git-Archeology:   Date: Wed, 16 Aug 2017 20:17:57 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: [WIP] Initial Rock64 support
> X-Git-Archeology:
> X-Git-Archeology: - Revision 9531d1bc7ecd0f468e29e402ba00cbc7b7dd683f: https://github.com/armbian/build/commit/9531d1bc7ecd0f468e29e402ba00cbc7b7dd683f
> X-Git-Archeology:   Date: Sat, 20 Jan 2018 18:58:33 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Fix rk3328 ATF build configuration
> X-Git-Archeology:
> X-Git-Archeology: - Revision 518ef98107a6705ba6649acdb1481cafedc9766d: https://github.com/armbian/build/commit/518ef98107a6705ba6649acdb1481cafedc9766d
> X-Git-Archeology:   Date: Fri, 16 Feb 2018 19:29:22 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: RK3399. More fixes but Odroid N1 refuses to boot :( ... ATF builds now (after apt install gcc-arm-none-eabi ?). Tested building/booting with 4-byte skip and u-boot with and without loading address. Enough for today.
> X-Git-Archeology:
> X-Git-Archeology: - Revision 8ce464f04930dc1589b7df6efdaa82c180e3c9b8: https://github.com/armbian/build/commit/8ce464f04930dc1589b7df6efdaa82c180e3c9b8
> X-Git-Archeology:   Date: Wed, 06 Nov 2019 18:33:43 +0000
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Rock Pi 4A fixed build with correct add-trust-ini.patch
> X-Git-Archeology:
> X-Git-Archeology: - Revision 91f2700eea18d9886fefa69d54bbbef44682083d: https://github.com/armbian/build/commit/91f2700eea18d9886fefa69d54bbbef44682083d
> X-Git-Archeology:   Date: Wed, 06 May 2020 23:45:09 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Fixes and cleanups after rockchip64 preparation for mainline u-boot (#1945)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 1c75b8fb91fccb8d58600ff18113ed04017fe3cb: https://github.com/armbian/build/commit/1c75b8fb91fccb8d58600ff18113ed04017fe3cb
> X-Git-Archeology:   Date: Fri, 12 Nov 2021 07:22:10 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Fix broken u-boot compilation on Rockpi S
> X-Git-Archeology:
> X-Git-Archeology: - Revision de26797423e22d58ec2882d7032c67f77196ecc5: https://github.com/armbian/build/commit/de26797423e22d58ec2882d7032c67f77196ecc5
> X-Git-Archeology:   Date: Sun, 06 Nov 2022 20:32:46 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move all legacy u-boot patches under one general legacy folder (#4386)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2813365dd25e3ad110936cbf014b95b38d7090ec: https://github.com/armbian/build/commit/2813365dd25e3ad110936cbf014b95b38d7090ec
> X-Git-Archeology:   Date: Mon, 07 Nov 2022 21:29:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move known non working rockhip64 boards to previous boot loader (#4392)
> X-Git-Archeology:
---
 trust.ini | 15 ++++++++++
 1 file changed, 15 insertions(+)

diff --git a/trust.ini b/trust.ini
new file mode 100644
index 0000000000..8cd7b97683
--- /dev/null
+++ b/trust.ini
@@ -0,0 +1,15 @@
+[VERSION]
+MAJOR=1
+MINOR=0
+[BL30_OPTION]
+SEC=0
+[BL31_OPTION]
+SEC=1
+PATH=bl31.elf
+ADDR=0x10000
+[BL32_OPTION]
+SEC=0
+[BL33_OPTION]
+SEC=0
+[OUTPUT]
+PATH=trust.bin
-- 
Armbian

